react/Viewer/PdfMobileViewer.spec.jsx

```diff
@@ -40,7 +41,9 @@ const setup = ({ file }) => {
   const root = render(
     <CozyProvider client={client}>
       <I18n lang="en" dictRequire={() => ''}>
-        <PdfMobileViewer file={file} t={x => x} />
+        <EncryptedProvider url={''}>
```

Comment:
nit: idem url string vide ?

Reply:
Folks, I don't disagree with any of this, but can we please stop being so "nitty"? It seems to me that the benefit/cost ratio of spotting/doing such changes is just not good: it brings no value, and the accumulation of such details is just too time-consuming, both for the reviewer and the reviewee.
There are already a lot of comments for this PR, and I did my best to address the most important ones. But it's just too much to handle if we have to go back and forth for every tiny detail: we'll just create friction and eventually accomplish nothing. I know that a nit is not mandatory to address, but I think we should avoid to focus so much on them and stick to the essential.
I hope this message is not too harsh, but I prefer expressing my concerns now, rather than later. I'm fully open to discuss about it if you wish :)

Reply:
ma question n'est pas sur la forme (ça n'a pas d'importance à mon sens d'écrire `url=""` ou `url={''}`), je me pose la question de pourquoi on est obligé de passer un string vide. Là j'interprète que ça plante avec un undefined 🤔 et que donc on a potentiellement oublié de gérer un cas. Le nit est une question, il ne nécessite pas forcément une modif du code.

La question est de savoir si mon interprétation est mauvaise, ou s'il y a un truc qui m'échappe ?

La PR est douloureuse c'est sûr, mais le Viewer est un composant délicat sur lequel on peut très vite avoir des régressions (d'ailleurs on en a une qui traine depuis février qu'on vient de découvrir ce matin) et qui a beaucoup de code legacy... de plus comme discuté on paye un peu le fait de ne pas avoir fait un grooming au préalable. Le coût avec cozy-ui est toujours un peu élevé malheureusement.

Maintenant la PR est validé, c'est du nit, donc si on est serein allons-y hein pas de souci.

Reply:
> je me pose la question de pourquoi on est obligé de passer un string vide. Là j'interprète que ça plante avec un undefined

Ok, donc non, ça ne plante pas, j'ai viré pour enlever toute ambiguité : [212d145](https://github.com/cozy/cozy-ui/pull/2112/commits/212d145fb7db151d80df8b7f199d50ebccceb476)

> le Viewer est un composant délicat sur lequel on peut très vite avoir des régressions 

Tout à fait, je ne reproche en rien la vigilance, plus la multiplication des nit que je ressens ces derniers temps sur les PR.

> de plus comme discuté on paye un peu le fait de ne pas avoir fait un grooming au préalable. Le coût avec cozy-ui est toujours un peu élevé malheureusement.

On fera mieux la prochaine fois ;)